packages/scripts/src/sitemaps/runner.ts

```diff
@@ -8,6 +8,12 @@ import {
 const log = logger(__filename);
 
 async function doExec() {
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
Noting this check actually isn't enough -> `NODE_ENV` production isn't limited to just production since it is used to determine logging/webpack/build (both explicitly by us but also internally by NodeJS/packages). As such, it can technically be used/tested locally. I've suggested this before with @kurtisassad that maybe we should have another variable for this purpose... I would suggest having an explicit variable such as `ENABLE_SITEMAP_GENERATOR` which we have to explicitly set to true in production and which will ensure that devs cannot accidentally run this script locally without being aware of the variable.

Reply:
It's actually because getThreadUrl will return localhost if not production:

<img width="781" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/45447/cc4923eb-cb7a-47f0-b3e6-1103e8a6c484">

Reply:
You bring up a valid point though as overwriting this wouldn't be a good thing but I really added it to avoid writing localhost URLs so I guess it's serving two use cases.

Reply:
Ah I see. Is there no scenario in which we would want to be able to generate a sitemap with localhost URLs for testing purposes?

In this case maybe we can use both `NODE_ENV` and and another variable to both ensure only `commonwealth.im` URLs and no localhost execution + overwriting.

Reply:
I'm fine with two but you could also screw this up by running the code in test and have a .env file in the local disk.

I think maybe a better solution could be to tighten up the .env permissions that we hand out?

Reply:
That's the point. Having a separate env var that we don't explicitly tell everyone that they need inherently tightens the execution whereas with just Node env anyone could accidentally run it. I'm for tightening as well. Might be able to better utilize Unleash for this kind of stuff.

Reply:
Like SITEMAP_ENV=production ? then abort otherwise?